### PR TITLE
(#3371) - 3.3.0 Release post

### PR DESCRIPTION
**Pull request by daleharvey**
@nolanlawson wanna give this a check?

### CODE REVIEW

File: docs/_posts/2015-02-03-fix-up-look-sharp.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+Nolan Lawson took on the mantle and has developed an implementation of Mango for PouchDB. It is currently in beta and you can try it out [as a plugin ](https://github.com/nolanlawson/pouchdb-find), try the [live demo](http://nolanlawson.github.io/pouchdb-find/) or even test it by installing [PouchDB-Server](https://github.com/pouchdb/pouchdb-server).
+
+### In other news
+
+The excellent team at [CozyCloud](https://www.cozycloud.cc/) have been working on a web based email application that you can run on a personal server based on PouchDB and ReactJS @ [https://www.npmjs.com/package/emails](https://www.npmjs.com/package/emails). Yours truly will be speaking about PouchDB and the offline web at [Web Rebels in Oslo](https://www.webrebels.org/speakers#daleharvey) and the [Hood.ie](http://hood.ie/) team are working on 1. [migrating their data storage to PouchDB](https://github.com/hoodiehq/wip-hoodie-store-on-pouchdb) and also [extracting some of PouchDB's test setup to be reusable](https://github.com/gr2m/testmate) contributions and discussion always welcome.
```
Comment (marten-de-vries) [inline docs/_posts/2015-02-03-fix-up-look-sharp.md]: The '1.' can be removed as far as I can see.

File: docs/_posts/2015-02-03-fix-up-look-sharp.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+title: PouchDB 3.3.0&#58; Fix Up, Look Sharp
+author: Dale Harvey
+
+---
+
+A new month means a new PouchDB release. Over the last month we have been looking around the rough edges and cleaning things up or chopping them off along with the usual slew of bugfixes and a lot of great news.
+
+### PouchDB-Inspector
+
+[Martin de Vries](https://twitter.com/commandoline) has taken the awesome work done by the CouchDB Fauxton team and wrapped fauxton up in Firefox and Chrome plugins so you inspect your local data in each browser.
```
Comment (marten-de-vries) [inline docs/_posts/2015-02-03-fix-up-look-sharp.md]: Marten de Vries (i -> e)

**Comment by marten-de-vries** [comment]
Except from the two nits I posted, LGTM. :)

**Comment by daleharvey** [comment]
Apologies, fixed your name in a follow up and nolan caught the other one

https://github.com/pouchdb/pouchdb/commit/b6237b3210b1f7026458c7a5f4daaade413e54ab

**Pull request closed without merging**